plugin/tests/phpunit/php/customizer/class-test-controls.php

```diff
@@ -629,20 +589,6 @@ public function test_get_frontend_css() {
 
 		// Assert we get the default values as CSS vars.
 		$this->assertContains( ':root {', $css );
-		$this->assertContains( '--mdc-theme-primary: #6200ee;', $css );
```

Comment:
This is removed since we don't have logic to generate color on PHP level - we get stored value from DB. Is that correct? @emeaguiar